apps/src/aiTutor/chatApi.ts

```diff
@@ -76,25 +84,25 @@ export async function getChatCompletionMessage(
       .logError('Error in chat completion request', error as Error);
   }
 
-  // For now, response will be null if there was an error.
-  if (!response) {
-    return {status: Status.ERROR};
-  } else if (response?.status === Status.PROFANITY_VIOLATION) {
-    return {
-      status: Status.PROFANITY_VIOLATION,
-      assistantResponse:
-        "I can't respond because your message is inappropriate. Please don't use profanity.",
-    };
-  } else if (response?.status === Status.PII_VIOLATION) {
-    return {
-      status: Status.PII_VIOLATION,
-      assistantResponse: `I can't respond because your message is inappropriate. Please don't include personal information like your ${response.status}.`,
-    };
+  if (!response) return {status: Status.ERROR};
+
+  switch (response.status) {
```

Comment:
This logic maps the `response.status` from the existing `ShareFilter` API (which is the type of violation) to one of two statuses `profanity_violation` or `pii_violation` we'll store in the db.

The previous checks would never have passed
```
response?.status === Status.PROFANITY_VIOLATION
```
and I switched to a `switch` statement because of the many-1 mapping of ShareFilter statuses to AITutorInteractionStatus.